src/styles/components/_chart.scss

```diff
@@ -125,7 +125,7 @@
   position: absolute;
   top: 0;
   pointer-events: none;
-  z-index: 999;
+  z-index: 998;
 }
 
 .chart-internal-tooltip {
```

Comment:
Ideally, this would be visibly hidden but there are a lot of charts currently in need of a rewrite.  For now, the z-index needs to be set a value less than the header (which is currently 999).